include/convex_bodies/vpolyintersectvpoly.h

```diff
@@ -322,6 +322,34 @@ class IntersectionOfVpoly {
     }
 
 
+    //------------------------------oracles for exponential sampling---------------//////
+
+    // compute intersection points of a ray starting from r and pointing to v
+    // with polytope discribed by A and b
+    std::pair<NT, int> quadratic_positive_intersect(Point const& r,
+                                    Point const& v,
+                                    VT const& Ac,
+                                    NT const& T,
+                                    VT& Ar,
+                                    VT& Av,
+                                    int& facet_prev) const
+    {
+        return std::make_pair(0, 0);
```

Comment:
Should it be better if you threw an `std::exception` here?

Reply:
This pollutes the `c++` code with `R` specific code, thus seems unclear. Why not throwing an exception here then catch it in the `Rcpp` part and throw an `Rcpp::exception(...)` there? 

It would be useful to add test cases that trigger those exceptions in both C++ (within a `try...catch...` block) and R.

This discussion triggers a question on how volesti handles errors in general.

Reply:
Thanks @vissarion. Yes I agree. I did what you suggested.